packs/feats/archetype/red-mantis-assassin/red-mantis-assassin-dedication.json

```
@@ -36,11 +36,75 @@
             "title": "Pathfinder Adventure: Prey for Death"
         },
         "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.Skill.Stealth",
+                        "predicate": [
+                            "skill:stealth:rank:0"
+                        ],
+                        "value": "trained"
+                    },
+                    {
+                        "label": "PF2E.Skill.Diplomacy",
+                        "predicate": [
+                            {
+                                "gte": [
+                                    "skill:stealth:rank",
+                                    1
+                                ]
+                            }
+                        ],
+                        "value": "expert"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Skill",
+                "rollOption": "red-mantis-assassin-stealth"
+            },
```

```suggestion
            {
                "adjustName": false,
                "choices": [
                    {
                        "label": "PF2E.ProficiencyLevel1",
                        "predicate": [
                            "skill:stealth:rank:0"
                        ],
                        "value": 1
                    },
                    {
                        "label": "PF2E.ProficiencyLevel2",
                        "predicate": [
                            {
                                "gte": [
                                    "skill:stealth:rank",
                                    1
                                ]
                            }
                        ],
                        "value": 2
                    }
                ],
                "flag": "rank"
                "key": "ChoiceSet",
                "prompt": "PF2E.SpecificRule.Prompt.Skill"
            },
```
Would this work instead? Then only have one upgrade AE like that reads the flag?

---

Yeah, good call